server/routes/sitemap.xml.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SitemapStream, streamToPromise } from 'sitemap'
+import { serverQueryContent } from '#content/server'
+// import appConfig from '~~/app.config'
+const appConfig = useAppConfig()
+
+export default defineEventHandler(async (event) => {
+  // Fetch all documents
+  const docs = await serverQueryContent(event).where({ _partial: false }).find()
```

Comment:
I totally forgot about draft pages. We also need to filter them out

```suggestion
  const docs = await serverQueryContent(event).where({ _partial: false, _draft: false }).find()
```